### ISSUE TITLE
Question ordering

### ISSUE THREAD

**Issue by itsrachelfish** [itsrachelfish/weightlifter#10]
Admins should be able to reorder questions from the questions list page, and maybe be able to edit a number from the edit question page?

**Comment by marcj303** [comment]
I think from the question list page would be most intuitive, but either or both would be fine.

**Comment by itsrachelfish** [comment]
I agree the question list page would be the most intuitive using drag & drop ([I wrote a library for this](https://www.npmjs.com/package/dragondrop)). But it might still be useful to let admins edit a number from the edit question page, like if someone wanted a question to always be last they could enter the number 999. The latter is more of a "nice to have" than anything else
